pages/js/jquery.magnific-popup.js

```diff
@@ -1,3 +1,1175 @@
\ No newline at end of file
+(function (a) {
+  'use strict'
+  typeof define === 'function' && define.amd
+    ? define(['jquery'], a)
+    : typeof exports === 'object'
+      ? a(require('jquery'))
+      : a(window.jQuery || window.Zepto)
+})(function (a) {
+  'use strict'
+  const b = 'Close'
+  const c = 'BeforeClose'
+  const d = 'AfterClose'
+  const e = 'BeforeAppend'
+  const f = 'MarkupParse'
+  const g = 'Open'
+  const h = 'Change'
+  const i = 'mfp'
+  const j = '.' + i
+  const k = 'mfp-ready'
+  const l = 'mfp-removing'
+  const m = 'mfp-prevent-close'
+  let n
+  const o = function () {}
+  const p = !!window.jQuery
+  let q
+  const r = a(window)
+  let s
+  let t
+  let u
+  let v
+  const w = function (a, b) {
+    'use strict'
+    n.ev.on(i + a + j, b)
+  }
+  const x = function (b, c, d, e) {
+    'use strict'
+    let f = document.createElement('div')
+    return (
+      (f.className = 'mfp-' + b),
+      d && (f.innerHTML = d),
+      e ? c && c.appendChild(f) : ((f = a(f)), c && f.appendTo(c)),
+      f
+    )
+  }
+  const y = function (b, c) {
+    'use strict'
+    n.ev.triggerHandler(i + b, c),
+    n.st.callbacks &&
+        ((b = b.charAt(0).toLowerCase() + b.slice(1)),
+        n.st.callbacks[b] &&
+          n.st.callbacks[b].apply(n, a.isArray(c) ? c : [c]))
+  }
+  const z = function (b) {
+    'use strict'
+    if (b !== v || !n.currTemplate.closeBtn) {
+      (n.currTemplate.closeBtn = a(
+        n.st.closeMarkup.replace('%title%', n.st.tClose)
+      )),
+      (v = b)
+    }
+    return n.currTemplate.closeBtn
+  }
+  const A = function () {
+    'use strict'
+    a.magnificPopup.instance ||
+      ((n = new o()), n.init(), (a.magnificPopup.instance = n))
+  }
+  const B = function () {
+    'use strict'
+    const a = document.createElement('p').style
+    const b = ['ms', 'O', 'Moz', 'Webkit']
+    if (a.transition !== undefined) return !0
+    while (b.length) if (b.pop() + 'Transition' in a) return !0
+    return !1
+  };
+  (o.prototype = {
+    constructor: o,
+    init: function () {
+      'use strict'
+      const b = navigator.appVersion;
+      (n.isLowIE = n.isIE8 = document.all && !document.addEventListener),
+      (n.isAndroid = /android/gi.test(b)),
```

Comment:
Codacy has a fix for the issue: [The 'g' flag is unnecessary because the regex is used only once in 'RegExp.prototype.test'.](https://app.codacy.com/gh/ruizivo/ahgora-login/pullRequest?prid=9873926)
```suggestion
      (n.isAndroid = /android/i.test(b)),
```